filestore.cabal

```diff
@@ -1,5 +1,5 @@
 Name:                filestore
-Version:             0.6.5
+Version:             0.6.6
```

Comment:
Hold off on incrementing the version. I will do this before the next release, taking account of the changes made.